From a628ece9b7e48d5b2b68b0e515d1c7bdd21f6bbc Mon Sep 17 00:00:00 2001
From: Kyle Manna <kyle@kylemanna.com>
Date: Sat, 29 Jul 2023 10:35:51 -0500
Subject: [PATCH 2/2] setup: Include ico and svg images

* Add package data.
* Needs to be a module to be installed.
---
 bmstools/gui/__init__.py     | 0
 bmstools/gui/img/__init__.py | 0
 setup.py                     | 2 ++
 3 files changed, 2 insertions(+)
 create mode 100644 bmstools/gui/__init__.py
 create mode 100644 bmstools/gui/img/__init__.py

diff --git a/bmstools/gui/__init__.py b/bmstools/gui/__init__.py
new file mode 100644
index 0000000..e69de29
diff --git a/bmstools/gui/img/__init__.py b/bmstools/gui/img/__init__.py
new file mode 100644
index 0000000..e69de29
diff --git a/setup.py b/setup.py
index 74ac08f..ab9dbed 100755
--- a/setup.py
+++ b/setup.py
@@ -48,6 +48,8 @@ setup(
     },
     license='Creative Commons Attribution-Noncommercial-Share Alike license',
     packages=find_packages(),
+    package_data={'': ['*.ico', '*.svg']},
+    include_package_data=True,
     entry_points = {
         'console_scripts': [
             'bmstools_jbd_gui=bmstools.gui.jbd_gui:main'
-- 
2.41.0

